src/app/welcome/index.njk

```diff
@@ -129,11 +129,12 @@ Create GAR | Start
 						To register for an event, please click on a link below:
 					</p>
 					<ul class="govuk-list govuk-list--bullet">
+						{#<li>No upcoming events.</li>#}
```

Comment:
Minor Change:  I think it is better to add a space before and after the list tags for readability.

Reply:
I will live with the current formatting, since we should get a linter sound, which will determine the code styles.